libs/view-transition/README.md

```diff
@@ -0,0 +1,62 @@
+# ![ng-web-apis logo](https://raw.githubusercontent.com/taiga-family/ng-web-apis/main/libs/view-transition/logo.svg) View Transition API for Angular
+
+[![npm version](https://img.shields.io/npm/v/@ng-web-apis/view-transition.svg)](https://npmjs.com/package/@ng-web-apis/view-transition)
+![npm bundle size](https://img.shields.io/bundlephobia/minzip/@ng-web-apis/view-transition)
+[![Coveralls github](https://img.shields.io/coveralls/github/ng-web-apis/view-transition)](https://coveralls.io/github/ng-web-apis/view-transition?branch=master)
+
+This service is an abstraction over
+[view transition API](https://developer.mozilla.org/en-US/docs/Web/API/View_Transitions_API) for Angular
+
+## Install
+
+```bash
+npm i @ng-web-apis/view-transition
+```
+
+## How to use
+
+1. Import the `ViewTransitionService` into your Angular component or service where you want to use it.
+
+```ts
+import {ViewTransitionService} from '@ng-web-apis/view-transition';
+```
+
+2. Inject the `ViewTransitionService` into your component's constructor or with `inject` (Angular 14+).
+
+```ts
+// in constructor
+constructor(private viewTransitionService: ViewTransitionService) {}
+
+// via inject
+viewTransitionService = inject(ViewTransitionService);
+```
+
+3. Use the `startViewTransition` method to initiate a view transition. This method takes a callback function that
+   returns a `Promise<void>` or `void`. You can perform any necessary view transition logic within this callback.
+
+```ts
+startTransition() {
+  this.viewTransitionService.startViewTransition(() => {
+    // Your view transition logic goes here
+    return this.animateTransition(); // Example: a transition animation function
+  }).subscribe(
+    (transition) => {
+      // Transition completed successfully
+      console.log('View transition completed:', transition);
```

Comment:
next is not completed, next is DOM changes were successful, complete is completed. Also I thought callbacks is deprecated signature of subscribe and you should use an object 🤔

Reply:
fixed